### PR TITLE
HOTT-1900 Two cols for the Rule of Origin tab + other improvements.

### PR DESCRIPTION
**Pull request by alexdesi**
### Jira link
https://transformuk.atlassian.net/browse/HOTT-1900

### What?
- [x] Make RoR tab two columns
- [x] remove section about "Product-specific rules"
- [x] Various wording improvements
- [ ] add section "How rules of origin could affect the import duty payable" (coming soo on HOTT-1902)

### CODE REVIEW

File: app/views/rules_of_origin/_tab.html.erb [hunk 1]
```diff
@@ -1,40 +1,55 @@
...
+                 commodity_code: commodity_code %>
+    <% end %>
+
+    <%#= render partial: 'rules_of_origin/scheme',
+               collection: rules_of_origin_schemes,
+               locals: {
+                 country_name: country_name,
+                 commodity_code: commodity_code,
+                 multiple_schemes: rules_of_origin_schemes.many?
+               } %>
```
Comment (alexdesi) [inline app/views/rules_of_origin/_tab.html.erb]: this can be removed

**Comment by alexdesi** [comment]
New RoR tab:
Screenshot 2022-08-23 at 16 22 17

File: app/views/rules_of_origin/_scheme.html.erb [hunk 2]
```diff
@@ -1,27 +0,0 @@
...
-      <span class="govuk-caption-l">
-        <%= scheme.title %>
-      </span>
-    <% end %>
-
-    Product-specific rules for commodity <%= commodity_code %>
-  </h3>
-
-  <%- if scheme.rules.any? -%>
-    <%= render 'rules_of_origin/rules_table', country_name: country_name,
```
Comment (willfish) [inline app/views/rules_of_origin/_scheme.html.erb:13]: I think we should add an explanation for why we're removing this to the PR

Comment (willfish) [inline app/views/rules_of_origin/_scheme.html.erb:13]: Removing it also means we've got a bunch of partials that are no longer used

File: app/views/rules_of_origin/_scheme.html.erb [hunk 3]
```diff
@@ -1,27 +0,0 @@
...
-                                              rules: scheme.rules %>
-
-    <%= render 'rules_of_origin/introductory_notes',
```
Comment (willfish) [inline app/views/rules_of_origin/_scheme.html.erb:16]: This partial can be removed

File: app/views/rules_of_origin/_scheme.html.erb [hunk 4]
```diff
@@ -1,27 +0,0 @@
...
-               introductory_notes: scheme.introductory_notes %>
-  <%- else -%>
-    <p>
-      There are no product-specific rules for commodity <%= commodity_code %>
-    </p>
-  <%- end -%>
-
-  <%= render 'rules_of_origin/proofs',
```
Comment (willfish) [inline app/views/rules_of_origin/_scheme.html.erb:24]: This partial can be removed

**Review by willfish — APPROVED** [review]

**Pull request merged**